mac

```diff
@@ -117,7 +117,7 @@ tap "caskroom/cask"
 # Unix
 brew "git"
 brew "openssl"
-brew "rcm"
+brew "rvm"
```

Comment:
Now that I took another look, I think we can remove this.  I'm 99.9% sure we've all switched to `rbenv`.

Reply:
Or replace it with rbenv if it doesn't already exist.

Reply:
rbenv is in the file. I haven't had to use rvm at all, so I went ahead and deleted it.